src/python-common/ceph/tests/test_service_spec.py

```diff
@@ -183,7 +191,7 @@ def test_yaml():
                                          service_type='mon',
                                          service_id='foo'
                                      ),
-                                     False
+                                     True
```

Comment:
ok that's scary. I'm pretty sure we have existing clusters with multiple MON specs applied already:

```yaml
service_type: mon
placement:
  count: 5
---
service_type: mon
service_id: mon
placement:
  hosts:
  - host1
  - host2
  - host3
```

we need to handle this case properly